src/main/java/cloud/eppo/api/Configuration.java

```diff
@@ -212,11 +195,12 @@ private static FlagConfigResponse parseFlagResponse(byte[] flagJson) {
         log.warn("Null or empty configuration string. Call `Configuration.Empty()` instead");
         return null;
       }
-      FlagConfigResponse config;
       try {
-        return mapper.readValue(flagJson, FlagConfigResponse.class);
-      } catch (IOException e) {
-        throw new RuntimeException(e);
+        return gson.fromJson(
+            new String(flagJson, StandardCharsets.UTF_8), FlagConfigResponse.class);
```

Comment:
So we're once again using automagic deserializers instead of explcit.

History lesson: we hand-rolled deserializers because the old auto magic relied on annotations, which got stripped away with Android's ProGuard minification and obfuscation. It seems _this_ automagic doesn't rely on annotations, but property names may change as well. Be sure to test this in the Android Dev studios with ProGuard on.